src/main/java/org/candlepin/hostedtest/HostedTestResource.java

```diff
@@ -266,7 +326,7 @@ public SubscriptionInfo getSubscription(@PathParam("subscription_id") String sub
      * Updates the specified subscription with the provided subscription data.
      *
      * @param subscriptionId the ID of the subscription to update
-     * @param subscriptionDTO
+     * @param subscription
```

Comment:
Missing createChildren parameter

Reply:
Fixed both; though my desire to just remove all of it from this class is growing.